flutter/integration_test/first_test.dart

```diff
@@ -70,16 +75,40 @@ void main() {
       await rm.init();
 
       final extendedResults = rm.getLastResult();
+
       final length = extendedResults.results.list.length;
 
       expect(length, expectedResultCount,
           reason:
               'results count should be $expectedResultCount, but it is $length');
 
+      final backendFilename =
+          extendedResults.results.list.first.backendInfo.filename;
+      expect(
+        expectedAccuracy[backendFilename],
+        isNotNull,
+        reason: 'missing accuracy expected values for $backendFilename',
+      );
+
       for (final benchmarkResult in extendedResults.results.list) {
         expect(benchmarkResult.performance, isNotNull);
         expect(benchmarkResult.performance!.throughput, isNotNull);
-        expect(benchmarkResult.accuracy, isNull);
+        expect(benchmarkResult.accuracy, isNotNull);
+        expect(benchmarkResult.accuracy!.accuracy, isNotNull);
+
+        final expectedAccuracyValue =
+            expectedAccuracy[backendFilename]![benchmarkResult.benchmarkId];
+        expect(
+          expectedAccuracyValue,
+          isNotNull,
+          reason:
+              'missing accuracy expected value for $backendFilename[${benchmarkResult.benchmarkId}]',
+        );
+        expect(
+          benchmarkResult.accuracy!.accuracy!.normalized,
+          greaterThanOrEqualTo(expectedAccuracyValue!),
```

Comment:
I would `expect` the accuracy to be in a range instead of `greaterThanOrEqualTo`. 
So maybe this in pseudo code:
```
tolerance  = 0.05
expect(realValue, greaterThanOrEqualTo(expectedValue * (1-tolerance)))
expect(realValue, lessThanOrEqualTo(expectedValue * (1+tolerance)))
```

Reply:
Done.
I have made it so you can specify expected range. Some benchmarks always produce the exact same result, and some have variations of different size.